### DISCUSSION TITLE
Can I simulate the ageing of a battery with PyBaMM?

### DISCUSSION THREAD

**Discussion by valentinsulzer** [pybamm-team/PyBaMM#1290] [Q&A] [answered]
A commonly asked question:
> I have tried with charge cycles between two given voltage thresholds,
but the discharge time never changed. I assume that I simply did not use
the right parameters for my simulations.

**Comment by valentinsulzer** [comment, marked as answer]
The default models (SPM, SPMe, DFN) do not include degradation mechanisms. To simulate degradation, you need to change the [model options](https://pybamm.readthedocs.io/en/latest/source/models/base_models/base_battery_model.html). 
At time of writing, we have models for:

- [SEI growth](https://github.com/pybamm-team/PyBaMM/tree/develop/pybamm/models/submodels/interface/sei)
- [Particle swelling and cracking](https://github.com/pybamm-team/PyBaMM/tree/develop/pybamm/models/submodels/particle_cracking)
- Lithium plating (almost! [PR in review](https://github.com/pybamm-team/PyBaMM/pull/1287))

Feel free to open an issue if you want to add a different degradation model. You can also look at the example scripts for calendar ageing and cycle ageing [here](https://github.com/pybamm-team/PyBaMM/tree/develop/examples/scripts)

**Reply by optibikash** [reply]
Thanks, Tino for the information. For adding the SEI growth model, the options are already given in the example file as:
options = {"sei": "ec reaction limited", "sei porosity change": "true"}
How do we add "Particle swelling and cracking" model in the simulation? 
Based on the source code, I think, we could plot particle crack length [m], Cell thickness change [m], particle surface tangential stress, and few others which I am not sure about. So basically, what are the parameters that we will be able to visualize at the end of the simulation in plots?
For "no cracking", an example is given in [here](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Validating_mechanical_models_Enertech_DFN.ipynb). Still, small documentation to explore other options would be really appreciated.

**Reply by valentinsulzer** [reply]
The options available are listed here: https://pybamm.readthedocs.io/en/latest/source/models/base_models/base_battery_model.html

**Reply by optibikash** [reply]
Thanks, and my apologies for the trouble. I was not aware of the documentation.
